.github/ISSUE_TEMPLATE/2 bug report.md

```diff
@@ -0,0 +1,5 @@
+---
+name: 🐛 Report bug
+about: __________________
+labels: 'bug'
+---
```

Comment:
I should probably turn this into an "issue form" and ***require** a repro URL as specified on:

https://github.com/sindresorhus/type-fest/issues/428